content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst

```diff
@@ -193,11 +193,32 @@ to simulate multiple packages. Products will need to be put in different :guilab
 too heavy. However, note that these rules will not apply to the shipping price calculation on the
 calculation on the sales order.
 
-When using a personal carrier contract
---------------------------------------
+Personal carrier contract
+-------------------------
 
-When using a personal carrier contract in Sendcloud, if the the price is not accurately reflected
-when creating a quotation in Odoo, then the pricing information needs to be updated in Sendcloud.
+To use custom contracts to determine shipping prices with a specific shipping carrier, log in to
+Sendcloud, go to :menuselection:`Settings --> Carriers --> My contracts`, and select the intended
+contract.
```

Comment:
The language here isn't clear for me and the sentence is a little long.
How about this?
```suggestion
Use custom prices from a direct carrier contract, via CSV upload, by first logging into Sendcloud,
navigating to :menuselection:`Settings --> Carriers --> My contracts`, and then selecting the
intended contract.
```